mappings/net/minecraft/world/chunk/ChunkDataList.mapping

```diff
@@ -0,0 +1,9 @@
+CLASS net/minecraft/class_5566 net/minecraft/world/chunk/ChunkDataList
```

Comment:
Data is already plural, would this make sense as just `ChunkData`?

Reply:
this class is so generic i hate it. `ChunkPositionData` maybe? it's literally a wrapper around a list with a `ChunkPos` attached

Reply:
It is data associated with a chunk, I think the name is probably fine? ChunkPositionData might imply that the data *is* the position, which may be a bit confusing?

Reply:
`ChunkData` implies a single blob of data. This is a list of blobs of data.